src/main/java/xyz/nucleoid/parties/PartyManager.java

```diff
@@ -179,6 +179,17 @@ public Party getParty(PlayerRef player) {
         return this.playerToParty.get(player);
     }
 
+    @Nullable
+    public Party getParty(UUID uuid) {
+        for (Party party : this.playerToParty.values()) {
+            if (uuid == party.getUuid()) {
```

Comment:
Hm, will this work? Or will it rather need to .equals()?

Reply:
Fixed in 57295c33496e22f0a5564099a437217d2b9dfdca